SpatialGDK/Source/SpatialGDK/Private/EngineClasses/SpatialNetDriver.cpp

```diff
@@ -1349,6 +1362,12 @@ void USpatialNetDriver::TickDispatch(float DeltaTime)
 			return;
 		}
 
+		if (bWaitForGlobalStateManagerServerTravelReady && IsGlobalStateManagerServerTravelReady())
```

Comment:
Let's change the `OnAcceptingPlayersChanged` to `OnGSMUpdated` or something similar.
In that function we can do a "bWaitingToSpawn" check and follow that with "CanSpawn" by checking the AcceptingPlayers and SessionID on the GSM. Then we can remove this additional logic from the TickDispatch as well as bWaitForGlobalStateManagerServerTravelReady and IsGlobalStateManagerServerTravelReady :D